source/css/scss/app/pages/monthly-good.scss

```diff
@@ -42,27 +42,14 @@
 			.mg-subhead {
 				text-align: left;
 				margin: 0;
-				padding-top: rem-calc(5);
-				font-size: rem-calc(20);
+				padding-top: rem-calc(15);
+				font-size: 1rem;
```

Comment:
no need to change this, but just for future reference, you could also use $base-font-size here